src/config/dynamic-theme-fixes.config

```diff
@@ -9177,13 +9177,20 @@ img[alt="GitHub Education"]
 education.yandex.ru
 
 CSS
-#fix-color__white .lc-text-block {
+[id^=dr-text-white_] .lc-text-block {
```

Comment:
Is it some fix for your personally created content?

Reply:
Yes